cosmos/dbt/graph.py

```diff
@@ -138,6 +163,11 @@ def parse_dbt_ls_output(project_path: Path | None, ls_stdout: str) -> dict[str,
                 file_path=project_path / node_dict["original_file_path"],
                 tags=node_dict.get("tags", []),
                 config=node_dict.get("config", {}),
+                has_freshness=(
+                    is_freshness_effective(node_dict.get("freshness", False))
+                    if node_dict["resource_type"] == DbtResourceType.SOURCE
+                    else False
+                ),
```

Comment:
It seems the output of `dbt ls` does not include the `freshness`  property :/

I cloned the [stacktonic-dbt-example-project](https://github.com/stacktonic-com/stacktonic-dbt-example-project)  repo, which has a source:
```
$ cat models/staging/google_analytics/src_google_analytics.yml
version: 2

sources:
  - name: google_analytics
    database: stacktonic-cloud # gcp-project
    dataset: analytics_263482362 # gcp-dataset
    
    freshness:
      warn_after: { count: 24, period: hour } 
      error_after: { count: 30, period: hour } 
    
    loaded_at_field: "parse_timestamp(_table_suffix, '%Y%m%d')"

    tables:
      - name: events
        identifier: events_*
        description: "Default Google Analytics 4 (raw) data dumps. Tables are date sharded (events_YYYYMMDD)."%    
``` 

I set up the local environment using:
```
python3.11 -m venv venv   
source venv/bin/activate
pip install dbt-core dbt-bigquery
cp .dbt/profiles.yml . 
```

And I run the following command using `dbt 1.4` and `dbt 1.8.3`:
```
 dbt ls --output json
```

Both of them generated an output that contained:
```
{
	"name": "events",
	"resource_type": "source",
	"package_name": "stacktonic_example_project",
	"original_file_path": "models/staging/google_analytics/src_google_analytics.yml",
	"unique_id": "source.stacktonic_example_project.google_analytics.events",
	"source_name": "google_analytics",
	"tags": [],
	"config": {
		"enabled": true
	}
}
```

So far, we have not relied on the `manifest.json` when using the dbt ls parsing mode, but it seems the time has come. We'll probably have to use it here.